Issue767 new pv system model

LauraMaier opened the pull request:
This is the PR for issue767.
A new pv system model was introduced which was validated using NIST measurement data.

The model offers a numerical and an analytical approach to estimate PV power generation and incorporates two different cell temperature calculation methods in order to take into account different mountings.

LauraMaier commented:
Things To Do before merging:

- [ ] Check if implemented model AixLib.BoundaryConditions.SolarGeometry.BaseClasses.DeclinationSpencer can be exchanged by existing model AixLib.BoundaryConditions.SolarGeometry.BaseClasses.Declination
- [ ] Check if model AixLib.BoundaryConditions.SolarGeometry.BaseClasses.IncidenceAngleDuffie can be exchanged by existing model AixLib.BoundaryConditions.SolarGeometry.BaseClasses.IncidenceAngle
- Transfer existing records to new format (if available):
- [ ] AleoS24
- [ ] CanadianSolarCS6P250P
- [ ] SchuecoSME1
- [ ] SymphonyEnergySE6M181

Dawac commented on AixLib/Electrical/PVSystem/BaseClasses/package.mo:
```diff
@@ -3,4 +3,140 @@ package BaseClasses
+
+  // Parameters from module data sheet
+
+   final parameter Modelica.SIunits.Efficiency eta_0=data.eta_0
```
How well can all of these parameters be extracted from datasheets?
Maybe the parameters in the record could be clustered to geometrical, electrical and general parameters for easier readability.

LauraMaier replied:
Done. The parameters are common manufacturer data. If data is not available default values were added for better usability :-)

Dawac reviewed (changes requested):
Comment: I would add Revision data to the models and records, that were modified within the process of remodelling the PV-models.

Changes: I commented for some models, that somehow only some new lines were randomly added, maybe these files don't need to be changed. One result file with unrelated results seems to have been deleated, please double check if this was on purpose.

General Questions:
- How were the existing records converted?
- Is the assumption of Tau*Alpha = 0,9 always reasonable, since its not adjustable. 
- Would it make sense to implement a "dirt-factor" or something like that?

LauraMaier commented on AixLib/Fluid/Actuators/package.mo:
```diff
@@ -32,14 +32,50 @@ package UsersGuide "User's Guide"
 
 
 
+
```
Merged development and these changes are ignored now ;-)

LauraMaier commented on AixLib/Media/AirIncompressible.mo:
```diff
@@ -391,7 +391,7 @@ algorithm
   This function computes the derivative of density with respect to
   temperature at constant pressure.
-</p>
+</p>/html /html
```
Same here. Solved by merging development

LauraMaier commented on AixLib/Media/Specialized/Water/ConstantProperties_pT/package.mo:
```diff
@@ -1,133 +1,145 @@
-          lineColor={238,46,47},
-          textString="pT")}));
-end ConstantProperties_pT;
+within AixLib.Media.Specialized.Water;
```
Solved by merge of development

LauraMaier commented on AixLib/DataBase/SolarElectric/AleoS24185.mo:
```diff
@@ -0,0 +1,44 @@
+</p>
+</html>",
+      revisions="<html><ul>
+  <li>
```
Done

LauraMaier commented on AixLib/Types/Tilt/package.mo:
```diff
@@ -1,36 +1,45 @@
-within AixLib.Types;
```
see resolved after merge

LauraMaier commented on AixLib/Resources/ReferenceResults/Dymola/AixLib_AirFlow_AirHandlingUnit_Examples_AHU.txt:
```diff
@@ -1,28 +0,0 @@
-last-generated=2019-03-27
```
Solved by merge of development

LauraMaier commented:
> Comment: I would add Revision data to the models and records, that were modified within the process of remodelling the PV-models.
> 
--> Changes done: added revision to all the models
> Changes: I commented for some models, that somehow only some new lines were randomly added, maybe these files don't need to be changed. One result file with unrelated results seems to have been deleated, please double check if this was on purpose.
> 
--> As discussed, this was partially caused by the html checker. To some parts, I revised it.

> General Questions:
> 
> * How were the existing records converted?
--> We converted them based on manufacturer data available online.

> * Is the assumption of Tau*Alpha = 0,9 always reasonable, since its not adjustable.
--> I introduced it as a final parameter for the first stage as it is not manufacturer data parameter, which users will know. It applies to Poly-Si-models with glass glazing.

> * Would it make sense to implement a "dirt-factor" or something like that?
--> As discussed, it would make sense in the future. Yet, this parameter really depends on the plant of interest and is determined rather random. Therefore, as a first step, I would like to not include it in the base model.

Dawac reviewed (approved):
Thanks for the effort! All issues have been adressed.
